waspc/cli/src/Wasp/Cli/Command/Telemetry/Project.hs

```diff
@@ -167,6 +179,7 @@ sendTelemetryData telemetryData = do
                   "wasp_version" .= _waspVersion telemetryData,
                   "os" .= _os telemetryData,
                   "is_build" .= _isBuild telemetryData,
+                  "deploy_cmd" .= _deployCmd telemetryData,
```

Comment:
Do we need to update posthog with anything before people start running this? Or is it effectively a schemaless system?

Reply:
Good point, can do